src/main/webapp/app/assessment-instructions/expandable-problem-statement/expandable-problem-statement.component.html

```diff
@@ -0,0 +1,14 @@
+<h5 class="expandable-header" (click)="isCollapsed = !isCollapsed">
+    {{ 'assessmentInstructions.problemStatement' | translate }}
+    <fa-icon [icon]="isCollapsed ? 'angle-right' : 'angle-down'"></fa-icon>
+</h5>
+<div [ngbCollapse]="isCollapsed">
+    <p *ngIf="exercise.type !== ExerciseType_PROGRAMMING; else programmingInstructions" [innerHTML]="exercise.problemStatement | htmlForMarkdown"></p>
+    <!-- Programming exercises have a couple of custom render modules, so we load the instructions component here.-->
+    <ng-template #programmingInstructions>
```

Comment:
We could work with content projection and extend the `jhi-expandable-paragraph` component to allow to pass alternate representations into the component. This would save us all the different representations we have for different assessment instructions. Also thinking of my changes in #1108.